package.json

```diff
@@ -1,52 +1,38 @@
 {
   "name": "react-docgen-annotation-resolver",
   "version": "1.1.0",
-  "main": "dist/index.js",
-  "module": "dist/index.js",
+  "main": "index.js",
   "repository": "git@github.com:Jmeyering/react-docgen-annotation-resolver.git",
   "author": "Jared Meyering <jared.meyering@gmail.com>",
   "license": "MIT",
+  "files": [
+    "index.js"
+  ],
   "scripts": {
-    "watch": "rimraf dist/ && babel src/ --out-dir dist/ --watch",
-    "build": "rimraf dist/ && babel src/ --out-dir dist/",
     "test": "jest",
-    "lint": "eslint './src/**/*.js*'"
+    "lint": "eslint '*.js'",
+    "prettier": "prettier --write '*.js'"
+  },
+  "engines": {
+    "node": ">=8.10.0"
   },
   "lint-staged": {
-    "*.{js,jsx,json}": [
-      "prettier --trailing-comma all --write",
+    "*.js": [
+      "npm run prettier",
```

Comment:
just run `prettier --write` here instead. Otherwise the `*.js` from the script going to render all benefits of lint-staged to zero

Reply:
I'd also suggest to run `eslint` here as well

Reply:
ok. thx.